### ISSUE TITLE
Silent failure 

### ISSUE THREAD

**Issue by darkjaguar** [carsonmcdonald/HTTP-Live-Video-Stream-Segmenter-and-Distributor#5]
Hi there,

I'm using the following command

> ./live_segmenter 10
> /var/www/ samp ./example-configs/config-simple.yml

nothing happens and the app is totally silent.

**Comment by dreist** [comment]
Same issue here

**Comment by darkjaguar** [comment]
I figured out how to use it, but had to read through all your blog posts. If it could return an error or warning it would help people use it. :) Thanks for your contribution though, very much appreciated!

**Comment by laurentnguyen-zz** [comment]
Dear Carson, 
I read with attention your website and I compiled yesterday the live_segmenter

Unfortunately as you will see below, the binary is not processing anything.

[root@teckel segmenter_new]# make all
gcc -Wall -g live_segmenter.c -o live_segmenter -lavformat -lavcodec -lavutil -lbz2 -lm -lz -lfaac -lmp3lame -lx264 -lfaad -lpthread

[root@teckel segmenter_new]# ./live_segmenter
Usage: ./live_segmenter <segment length> <output location> <filename prefix> <encoding profile>

[root@teckel segmenter_new]# ./live_segmenter 20 /home/laurent/segmenter_new/riena.mpeg riena ep_128k

Here is the detail of my compiled ffmpeg (Red Hat 5.3 on AMD 64 bits)

[root@teckel ffmpeg]# ./ffmpeg
FFmpeg version SVN-r26402, Copyright (c) 2000-2011 the FFmpeg developers
  built on Feb  7 2011 06:54:39 with gcc 4.1.2 20080704 (Red Hat 4.1.2-50)
  configuration: --enable-gpl --enable-nonfree --enable-pthreads --enable-libfaac --enable-libmp3lame --enable-libx264
  libavutil     50.36. 0 / 50.36. 0
  libavcore      0.16. 1 /  0.16. 1
  libavcodec    52.108. 0 / 52.108. 0
  libavformat   52.93. 0 / 52.93. 0
  libavdevice   52. 2. 3 / 52. 2. 3
  libavfilter    1.74. 0 /  1.74. 0
  libswscale     0.12. 0 /  0.12. 0
Hyper fast Audio and Video encoder

Kind Regards,
Laurent

**Comment by ePirat** [comment]
Yeah there is a ffmpeg but that prevents the segmenter from working...
